common/species_rights/zz_hfe_vanilla_replace_species_rights.txt

```diff
@@ -57,21 +60,62 @@ citizenship_slavery = {
 		custom_tooltip = {
 			fail_text = does_not_embrace_species_slavery
 			# mod
-			or = {
-				has_species_flag = hfe_forever_slave
-				has_species_flag = hfe_forever_slave@from
-				has_trait = hfe_species_trait_happy_slave
-				from = {
+			OR = {
```

Comment:
This was a better way to word it and prevented weird things like Forever Slave Ahets from being enslaved by Egalitarians or Xenophiles that are not the Ju Ahets. In short, the Forever Slaves are meant to be slaves for the Ju Ahet and nations that actually have slaves.

This was a quality improvement to the entire thing. In my opinion, this shouldn't have existed because of the fixes I made to the Ju Ahets enslaving Xenoes.